### ISSUE TITLE
webpack dev server support

### ISSUE THREAD

**Issue by volarname** [cascornelissen/svg-spritemap-webpack-plugin#192]
First of all thank you for this great plugin, i tried several one and this one is best working, easy to configure and most up to date.

**Description**
Does it support webpack dev server?

When i have a config with direct files in array like this:
```js
new SVGSpritemapPlugin(
        [
          'src/svgSprite/**/*.svg',
          '../common/src/svgSprite/arrow-up-1.svg',
          '../common/src/svgSprite/fb.svg',
          '../common/src/svgSprite/chevron-right.svg',
          '../common/src/svgSprite/cross-solid.svg',
          '../common/src/svgSprite/cross.svg',
          '../common/src/svgSprite/instagram.svg',
        ],
        {
          output: {
            filename: 'images/svg-sprite.svg',
          },
          sprite: {
            prefix: 'svg-icon-',
            generate: {
              title: false,
            },
          },
          styles: false,
        }
      ),
```
when i start webpack dev server i get errors in cli:
```
Watchpack Error (initial scan): Error: ENOTDIR: not a directory, scandir '/var/www/html/common/src/svgSprite/arrow-up-1.svg'
Watchpack Error (initial scan): Error: ENOTDIR: not a directory, scandir '/var/www/html/common/src/svgSprite/fb.svg'
Watchpack Error (initial scan): Error: ENOTDIR: not a directory, scandir '/var/www/html/common/src/svgSprite/chevron-right.svg'
Watchpack Error (initial scan): Error: ENOTDIR: not a directory, scandir '/var/www/html/common/src/svgSprite/cross-solid.svg'
Watchpack Error (initial scan): Error: ENOTDIR: not a directory, scandir '/var/www/html/common/src/svgSprite/cross.svg'
```
Everything works when i build the project, just watch for changes using webpack dev server is problem.
So my question is, does it support webpack dev server? if yes, its a bug? if no how about to add feature to support it?

Thank you

**Comment by cascornelissen** [comment]
First time I'm seeing someone point to specific files 🤔 This could probably be implemented and tested, shouldn't be too hard. The assumption is made here that each input pattern is a directory it seems so we'd need some additional checks there.

https://github.com/cascornelissen/svg-spritemap-webpack-plugin/blob/46d9fd4f081ba4d73bc0fbe6f11ea421648c6a21/lib/index.js#L370

Feel free to submit a PR!

**Comment by volarname** [comment]
our use case is that we have shared folder with svg files and then we have project specific svg files, so our svg files are not duplicated. we build sprite for each project individually, so we need to reference individual files

**Comment by cascornelissen** [comment]
I understand and it makes sense, just don't think I've seen anyone before with that use-case. As said, a PR is welcome or I'll look into it myself at some point in the future if I have some time.

**Comment by volarname** [comment]
take a look, feel free to comment:
https://github.com/cascornelissen/svg-spritemap-webpack-plugin/pull/193

**Comment by cascornelissen** [comment]
I've made some changes in 055d348 that should solve this. Could you try it out on your end by installing directly from `master` by running the following command?

```shell
npm install cascornelissen/svg-spritemap-webpack-plugin#master
```

**Comment by volarname** [comment]
> I've made some changes in [055d348](https://github.com/cascornelissen/svg-spritemap-webpack-plugin/commit/055d34846d2ccdf8b8423301580ee124cc8b01f4) that should solve this. Could you try it out on your end by installing directly from `master` by running the following command?
> 
> ```shell
> npm install cascornelissen/svg-spritemap-webpack-plugin#master
> ```

i can confirm that this version works without any warning/error and svg sprite file is correctly available in dev server. thank you.
when it will land in a release or hotfix officially?

**Comment by cascornelissen** [comment]
That's great to hear, thanks for the quick reply! I'm hoping to release this together with the fix for #194 but if that takes too long I'll make sure to push out a release ✌🏼

**Comment by cascornelissen** [comment]
Just got confirmation the other issue is also solved so I've published version [`4.4.1`](https://github.com/cascornelissen/svg-spritemap-webpack-plugin/releases/tag/v4.4.1) which includes this fix.
